src/main/java/org/folio/innreach/batch/contribution/service/CustomAckMessageListener.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.innreach.batch.contribution.service;
+
+import lombok.AllArgsConstructor;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.folio.innreach.batch.contribution.ContributionJobContext;
+import org.folio.innreach.domain.dto.folio.inventorystorage.InstanceIterationEvent;
+import org.springframework.kafka.listener.AcknowledgingMessageListener;
+import org.springframework.kafka.support.Acknowledgment;
+
+@AllArgsConstructor
+public class CustomAckMessageListener implements AcknowledgingMessageListener<String, InstanceIterationEvent> {
```

Comment:
I think this is no longer an "acknowledging message listener" because it no longer acks (commits). So I think two things should happen here:
* Go back to the sample code. It had an implementation that wasn't acknowledging and implement that instead here. Of course testing that it works.
* Rename this class to something more meaningfor our purposes like `InitialContributionMessageListener`.

Reply:
Don't think this is resolved Arin. Should this be an `AcknowledgingMessageListener` or just a `MessageListener` implementation. [This](https://github.com/bkatwal/kafka-util/blob/master/src/main/java/com/bkatwal/kafka/impl/CustomMessageListener.java#L10)is the sample code I'm referring to.

On line 26 you're no longer acknowledging. Because we have retry, we don't need to commit/ack, right? It's auto committing now, and retry is smart enough to do things correctly.